srcpkgs/walk/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'walk'
+pkgname=walk
+version=1.0.0
+revision=1
+build_style=gnu-makefile
+hostmakedepends="gzip make"
+short_desc="Two utilities collectively replacing find: walk and sor"
+maintainer="Andrea D'Amore <a@d-amo.re>"
+license="Apache-2.0"
+homepage="https://git.benjamin.barenblat.name/walk/"
+distfiles="https://github.com/bbarenblat/${pkgname}/archive/refs/tags/v${version}.tar.gz"
+checksum=7d733cc106ca81390197f772efd4243ab798fe12ee7de30ceec3ce92cb762375
+
+pre_install() {
+	make_install_args="prefix=${DESTDIR}/usr"
```

Comment:
can this line be in the main block of variables - outside of this function?

Reply:
Maybe. But I wasn't able to build on ARM archs while setting make_install_args at root level since `DESTDIR` embeds the cross-compiler into the path and I could figure how to access it at root level.

Reply:
> Possibly by quoting special character `$` like in `srcpkgs/9mount/template`.

Oh, right. I see why that was not available.

I will single quote the string to prevent expansion, then.

Thanks.

Reply:
Bump, can this be done?